### PR TITLE
Pal 393 remove cache service

### PR DESCRIPTION
**Pull request by dev930018**

### CODE REVIEW

File: data-service/pom.xml [hunk 1]
```diff
@@ -30,51 +30,16 @@
```
Comment (pd104923) [inline data-service/pom.xml:30]: The copyright should be updated due to the number or deletions.

File: palisade-service/pom.xml [hunk 2]
```diff
@@ -30,45 +30,16 @@
```
Comment (pd104923) [inline palisade-service/pom.xml]: The copyright should be updated due to the number or deletions.

File: resource-service/pom.xml [hunk 3]
```diff
@@ -31,9 +31,6 @@
     <properties>
```
Comment (pd104923) [inline resource-service/pom.xml:31]: The copyright should be updated due to the number or deletions.

File: resource-service/src/main/java/uk/gov/gchq/palisade/service/resource/config/ApplicationConfiguration.java [hunk 4]
```diff
@@ -29,25 +28,13 @@
```
Comment (pd104923) [inline resource-service/src/main/java/uk/gov/gchq/palisade/service/resource/config/ApplicationConfiguration.java:28]: The copyright should be updated due to the number or deletions.

**Review by pd104923 — COMMENTED** [review]
Just a few copyright updates for some pom files as they have been changed quite a bit.

**Comment by ac74475** [comment]
closing until all the services have finished being moved to spring caching

**Review by developer01189998819991197253 — APPROVED** [review]

**Review by ac74475 — APPROVED** [review]

**Pull request merged**